charts/csi-baremetal-deployment/values.yaml

```diff
@@ -79,6 +79,18 @@ driver:
     image:
       # if you want to use topology feature (multiple PVCs per pod) you should use v1.2.2
       tag: v3.0.0
+    envVars:
+      # the following parameters is required https://github.com/kubernetes-csi/external-provisioner#command-line-options
+      - name: ADDRESS
+        value: "/csi/csi.sock"
+      - name: TIMEOUT_DURATION
+        value: 30s
+      - name: RETRY_INTERVAL_START
```

Comment:
Could we comment envs here and pass the related argument in the controller, if env variable is set only? So we will keep default provisioner variables, if they are changed from version to version. Or is it redundant?

Reply:
That values holds default parameters

Reply:
I mean to not pass the related arg, if env is excluded. For example:
1. In helm values
```
      - name: TIMEOUT_DURATION
        value: 30s
     #- name: RETRY_INTERVAL_START
     #   value: 1s
```
2. Args
```
["--timeout=$(TIMEOUT_DURATION)",]
```
And no `"--retry-interval-start=$(RETRY_INTERVAL_START)",`

So default provisioner timeout will be used

Reply:
It's kind complex I think. If you omit some args then default values will be used under the hood. I've investigate another way by passing args but found env variables a more flexible solutions.

Reply:
Got it

Reply:
You point me to good idea 👍 , I'll refactor my code.